Source/Scenes/0_Onboarding/1_AskForAnalyticsPermissions/AskForAnalyticsPermissionsViewModel.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  AskForAnalyticsPermissionsViewModel.swift
+//  Zupreme
+//
+//  Created by Andrei Radulescu on 09/11/2018.
+//  Copyright © 2018 Open Zesame. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+
+// MARK: - AnalyticsPermissionNavigation
+enum AskForAnalyticsPermissionsNavigation: TrackedUserAction {
+    case userOptedForAnalyticsPermissions
```

Comment:
these cases should be named as if the where prefixed with `userDid` so change this to `optForAnalytics` or `answerQuestionAboutAnalyticsPermission` or similar.

the Coordinator names the step `userDid` and the ViewModel where we step to the step has the method `userDid`  :)